cmd/test_server/main.go

```diff
@@ -23,13 +24,18 @@ func impl(args []string, in io.Reader, out, errs io.Writer, exit func(int)) {
 	app.Terminate(exit)
 
 	port := app.Flag("port", "The port to run on").Default("7002").Int()
+	useFakeTime := app.Flag("fake-time", "Use a fake time for more stable commits hashes").Default("false").Bool()
```

Comment:
Since this is a test server should we just always use the fake time? Or are you thinking about the dev setup? Alternately, we need to update the HACKING.md doc.

Reply:
Let's default it to true then.